### PR TITLE
Url to uri

### PR DESCRIPTION
**Pull request by kunsingh**
Changed all occurrence of URL to URI

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/780)

### CODE REVIEW

File: platform-examples/distribution-sample/distribution-client/src/main/java/com/canoo/platform/samples/distribution/client/DistributionClient.java [hunk 1]
```diff
@@ -28,13 +28,17 @@
 import javafx.scene.layout.VBox;
 import javafx.stage.Stage;
 
+import java.net.URISyntaxException;
 import java.net.URL;
 
 public class DistributionClient extends Application {
 
     private void connect(final Stage primaryStage, final URL endpoint) {
```
Comment (hendrikebbers) [inline platform-examples/distribution-sample/distribution-client/src/main/java/com/canoo/platform/samples/distribution/client/DistributionClient.java]: why not "final URI endpoint" as param?

Comment (kunsingh) [reply platform-examples/distribution-sample/distribution-client/src/main/java/com/canoo/platform/samples/distribution/client/DistributionClient.java]: Done.

**Comment by hendrikebbers** [comment]
@kunsingh You need to merge the master branch in this branch.

**Comment by hendrikebbers** [comment]
fix for https://github.com/canoo/dolphin-platform/issues/714

**Pull request merged**